soh/soh/Enhancements/randomizer/context.cpp

```diff
@@ -140,7 +140,7 @@ void Context::GenerateLocationPool() {
     if (mSettings->GetOption(RSK_TRIFORCE_HUNT)) {
         AddLocation(RC_TRIFORCE_COMPLETED);
     }
-    AddLocations(StaticData::overworldLocations);
+    AddLocations(StaticData::GetOverworldLocations());
```

Comment:
This function erroneously includes RC_TRIFORCE_COMPLETED in it's search, causing it to be added when triforce hunt is off, causing seeds to fail to generate without it if entrance rando is on due to the stricter AllLocationsAccessible check.